os_info/src/linux/file_release.rs

```diff
@@ -144,6 +145,18 @@ mod tests {
         assert_eq!(info.codename, None);
     }
 
+    #[test]
+    fn os_release_rhel() {
+        let mut distributions = [DISTRIBUTIONS[0].clone()];
+        distributions[0].path = "src/linux/tests/os-release-rhel";
```

Comment:
Change that to this?  (*rhel* is upsetting the spellchecker)
```Rust
        distributions[0].path = "src/linux/tests/os-release-rhel";  // cspell:disable-line
```

Reply:
Ah.  Got it...
https://github.com/stanislav-tkach/os_info/blob/master/os_info/src/windows/winapi.rs#L1-L3